examples/readers/tensorflow_data_api_tiledb_dense.ipynb

```diff
@@ -11,7 +11,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```

Comment:
This notebook looks it hasn't run?

Reply:
Hmm maybe not overwritten. Let me check again.